exporter/kafkaexporter/kafka_exporter_test.go

```diff
@@ -78,6 +82,7 @@ func TestTraceDataPusher_marshall_error(t *testing.T) {
 	}
 	td := testdata.GenerateTraceDataTwoSpansSameResource()
 	droppedSpans, err := p.traceDataPusher(context.Background(), td)
+	require.Error(t, err)
```

Comment:
seems not needed as it would fail on the assert below

Reply:
The next line can produce null pointer exception, so it is recommended to check for not nil first.